test-data/unit/check-ctypes.test

```diff
@@ -167,4 +167,16 @@ reveal_type(intarr4(*int_values))  # E: Revealed type is 'ctypes.Array[ctypes.c_
 reveal_type(intarr4(*c_int_values))  # E: Revealed type is 'ctypes.Array[ctypes.c_int]'
 reveal_type(intarr6(1, ctypes.c_int(2), *int_values))  # E: Revealed type is 'ctypes.Array[ctypes.c_int]'
 reveal_type(intarr6(1, ctypes.c_int(2), *c_int_values))  # E: Revealed type is 'ctypes.Array[ctypes.c_int]'
+
+float_values = [1.0, 2.0, 3.0, 4.0]
+intarr4(*float_values) # E: Array constructor argument 1 of type "builtins.list[builtins.float*]" is not convertible to the array element type "Iterable[ctypes.c_int]"
+[builtins fixtures/floatdict.pyi]
+
+[case testCtypesArrayConstructorKwargs]
+import ctypes
+intarr4 = ctypes.c_int * 4
+
```

Comment:
I wanted to add a `intarr4(1, 2, 3, x=4)` test case here, but not sure how to get the tests to pass: the output on my machine is:

```
Expected:
  main:4: error: Unexpected keyword argument "x" for "Array"
  main:7: error: Too many arguments for "Array" (diff)
Actual:
  main:4: error: Unexpected keyword argument "x" for "Array"
  /home/alan/workspace/python/mypy/mypy/typeshed/stdlib/2and3/ctypes/__init__.pyi:273: note: "Array" defined here (diff)
  main:7: error: Too many arguments for "Array" (diff)
```

I added a `# E: ` comment for the error, but don't know how to handle the note (which includes the path, so will be setup specific). Thoughts?

Reply:
Oh, this note caused some problems before. One option would be to add some very simple mock stubs to `test-data/unit/lib-stub`. But probably even better is just not add this test. The two tests you have added are sufficient.